### DISCUSSION TITLE
Json Structure

### DISCUSSION THREAD

**Discussion by NX115** [Varelion/Npower_Cloud_Study_Flashcards#2] [Q&A] [answered]
Hey, what's the Json structure we're using?

**Comment by Varelion** [comment, marked as answer]
`[
	{
		"question": "Sample question lorem ipsum?",
		"correctAnswersArray": [
			"Sample answerlorem ipsum", 
			"another possible answer, as you can have many, lorem ipsum"
		],
		"wrongAnswersArray": [
			"Sample wrong question lorem ipsum",
			"Sample wrong question lorem ipsum",
			"Sample wrong question lorem ipsum"
		]
	},
]`

However, this will change to include additional fields when you export it, based on the difficulty rating you give a question, the notes, or hints you type down to help cement topics and clarifications. Always export your file, and save it as a `whatever_file_name.json`, and then CTRL+A, CTRL+C to copy all the content, and paste it into the importer with CTRL+V.

**Comment by NX115** [comment]
Oh, right! I can see it in the code now, thank you!
